pkg/tfgen/generate_python.go

```diff
@@ -794,6 +793,82 @@ func (g *pythonGenerator) recordPropertyRec(sch *schema.Schema, info *tfbridge.S
 	}
 }
 
+// emitMembers emits property declarations and docstrings for all output properties of the given resource.
+func (g *pythonGenerator) emitMembers(w *tools.GenWriter, mod *module, res *resourceType) {
+	for _, prop := range res.outprops {
+		name := pyName(prop.name)
+		ty := pyType(prop)
+		w.Writefmtln("    %s: pulumi.Output[%s]", name, ty)
+		if prop.doc != "" {
+			g.emitDocComment(w, prop.doc, "    ")
+		}
```

Comment:
I had no idea this was a standard format for property documentation - and couldn't find any examples of this in the wild (and VS Code doesn't seem to pick up the doc strings?).  But I assume this works for Sphinx docgen?

Reply:
It does. This is the standard Sphinx autodoc form. There are a few other forms out there (NumPy and Google both have their own forms) but as far as I can tell this is "the standard".

The fact that VSCode's Python extension doesn't handle this well seems to be well-documented: https://github.com/Microsoft/vscode-python/issues/38. It's pretty unfortunate, if this issue is still relevant.

Reply:
Based on the preponderance of tensorflow/scipy, I do wonder if numpy/google's style is "winning", but at this point all we really need to do is generate something that Sphinx can understand, and Sphinx can understand any of those with the right config.

Reply:
Yep - makes sense - totally fine with this as-is.